pytorch_lightning/__init__.py

```diff
@@ -13,12 +13,13 @@
     _logger.addHandler(logging.StreamHandler())
     _logger.propagate = False
 
+from pytorch_lightning.accelerators import Accelerator  # noqa: E402
 from pytorch_lightning.callbacks import Callback  # noqa: E402
 from pytorch_lightning.core import LightningDataModule, LightningModule  # noqa: E402
 from pytorch_lightning.trainer import Trainer  # noqa: E402
 from pytorch_lightning.utilities.seed import seed_everything  # noqa: E402
 
-__all__ = ["Trainer", "LightningDataModule", "LightningModule", "Callback", "seed_everything"]
+__all__ = ["Accelerator", "Trainer", "LightningDataModule", "LightningModule", "Callback", "seed_everything"]
```

Comment:
Why this change?

Reply:
to annotate the types in the plugins with "pl.Accelerator". Otherwise there appear to be circular imports unfortunately. Haven't found a different solution yet.

Reply:
so annotating with `"pl.accelerators.accelerator.Accelerator"` doesn't work? This might be a bit longer, but since we don't import it directly, that should be fine I think

Reply:
`__all__` is just to indicate what's imported with `from pytorch_lightning import *`

It shouldn't relate to circular imports at all

Reply:
okay whatever works